pkg/kubelet/prober/prober.go

```diff
@@ -112,6 +112,9 @@ func (pb *prober) probe(ctx context.Context, probeType probeType, pod *v1.Pod, s
 	if result == probe.Warning {
 		pb.recordContainerEvent(pod, &container, v1.EventTypeWarning, events.ContainerProbeWarning, "%s probe warning: %s", probeType, output)
 		klog.V(3).InfoS("Probe succeeded with a warning", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name, "output", output)
+	} else if result == probe.Unknown {
+		klog.V(3).InfoS("Probe succeeded with a unknown state", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
```

Comment:
Any reason to log before recording an event? If so, please change it for probe.Warning as well.

Reply:
I believe there's no specify reason to log before recording an event, but since the former code was log before an event, I would prefer to follow it up

Reply:
<img width="604" alt="image" src="https://user-images.githubusercontent.com/73322208/230587051-b2de72ed-f16b-4e44-8c3d-75cfc01e2525.png">

I double check the git commit